bazel/thirdparty/openssl.BUILD

```diff
@@ -32,6 +39,8 @@ filegroup(
 
 configure_make(
     name = "openssl",
+    # These don't get make variables expanded, so use the injected environment variable.
+    args = ["-j$OPENSSL_BUILD_JOBS"],
```

Comment:
seems like foreign_cc rules uses expand_location on args, but not expand_make_variables, which i'm guessing is why we have to jump for the environment variable hoop here?

Reply:
Yes exactly. I think there is an issue open for this... But the make/cmake rules are very inconsistent about this.